a bit of an ugly hack to import the module properly

`from sphinxcontrib ...` doesn't work, because sphinxcontrib is defined by
all the other py3-sphinxcontrib-* packages that use (now depreacated) declare_namespace
as a namespace residing in sitedir, thus python doesn't see the local directory

diff --git a/setup.py b/setup.py
index b0bf058..e43693e 100644
--- a/setup.py
+++ b/setup.py
@@ -1,5 +1,8 @@
 #-*- coding:utf-8 -*-
 
+import sys
+del sys.modules['sphinxcontrib']
+
 import setuptools
 from sphinxcontrib import gist as pkg
 
